ome_zarr/scale.py

```diff
@@ -125,6 +130,30 @@ def __create_group(
             series.append({"path": path})
         return grp
 
+    def resize_image(self, image: ArrayLike) -> ArrayLike:
```

Comment:
I guess that none of these issues is a blocker - Improving the `Scaler` documentation and usage - Using the `scaler.resize_image()` instead of `scaler.nearest()` for dask AND non-dask data, possibly removing other methods from the class etc could come in a follow-up PR (even a follow-up release). cc @joshmoore?

Probably the most important fix should be handling `write_image(dask_data, group, scaler=None)` which I think would currently fail.

Reply:
I would agree that dask versions of gaussian, laplacian, etc could be done in later PRs.
I can try to do one more pass this afternoon to make sure `scaler=None` does not break with dask array. 
As I'm sure is true with many others, I am splitting my time among many different projects, so thanks for living with these intermittent pieces of work on this PR.

Reply:
Had a quick look and I am already handling `scaler=None` in `_write_dask_image`: (1) max_layer set to 0 if scaler is None, and (2) don't call scaler.resize_image if scaler is None